google/cloud/aiplatform/prediction/local_model.py

```diff
@@ -199,8 +201,23 @@ def create_cpr_model(
             |   |-- custom_package.tar.gz
             |   |-- ...
             |-- ...
+            |-- cpr_model_server.py
             |-- entrypoint.py
 
+        The created CPR images default the number of workers to the number of cores. If
```

Comment:
```suggestion
        The created CPR images default the number of model server workers to the number of cores. Depending on the characteristics of your model, you may need to adjust the number of workers. You can set the number of workers with the following environment variables:
            WORKERS_PER_CORE:
                The number of the workers per core. The default is 1.
            MAX_WORKERS:
                The maximum number of workers can be used given the value of WORKERS_PER_CORE
                and the number of cores.
            WEB_CONCURRENCY:
                The number of the workers. This will overwrite the number of workers calculated by
                the other variables, min(WORKERS_PER_CORE * number_of_cores, MAX_WORKERS).
        See https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker/blob/master/README.md#workers_per_core.
```